mithril-explorer/package-lock.json

```diff
@@ -36,9 +36,17 @@
     },
     "../mithril-client-wasm/pkg": {
       "name": "mithril-client-wasm",
-      "version": "0.5.1",
+      "version": "0.5.3",
```

Comment:
The library `mithril-client-wasm` version doesn't match the version specified in the `mithril-client-wasm` crate's [Cargo.toml](https://github.com/input-output-hk/mithril/pull/2004/files#diff-72ad6e1c1356d0f73e50d9e7970bd2defe29e80296db55ad0c0ad44362a6ea84R3).

It seems that a local compilation of `mithril-client-wasm` crate is missing in the runbook and the script before updating the explorer.